ospf6d/ospf6_top.c

```diff
@@ -217,6 +217,7 @@ void ospf6_delete(struct ospf6 *o)
 	route_table_finish(o->distance_table);
 
 	XFREE(MTYPE_OSPF6_TOP, o);
+	ospf6 = NULL;
```

Comment:
global struct should not be set to NULL in a common api. In this file caller already sets it to NULL. why set within the api? Instead make symmetric and call similarly in ospf6_main as well.

 349                 ospf6_delete(ospf6);
 350                 ospf6 = NULL;

Reply:
ok.